### PR TITLE
Loderunner checkov yaml fixes

### PR DESCRIPTION
**Pull request by AAkindele**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

- Fix checkov yaml issues in loderunner deployment template.

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- References https://github.com/retaildevcrews/wcnp/issues/637

### CODE REVIEW

**Comment by kforeverisback** [comment]
We tested the access policy of emptyDir and found that its strictly tied to each pod instance.
See [this description](https://github.com/retaildevcrews/ngsa-app/pull/101) for details

**Review by sivamu — CHANGES_REQUESTED** [review]
Can we test the YAML changes in the dev environment before deploying to preprod?

**Comment by sivamu** [comment]
I think the equivalent LR yaml file in the dev environment is: https://github.com/retaildevcrews/loderunner/blob/main/autogitops/dev/loderunner-commandmode.yaml

File: autogitops/preprod/loderunner.yaml [hunk 1]
```diff
@@ -32,6 +40,14 @@ spec:
             requests:
               memory: "512Mi"
               cpu: "500m"
+          livenessProbe:
```
[the remark this answers is not in the document]
Comment (AAkindele) [reply autogitops/preprod/loderunner.yaml:43]: I double checked and there is no healthz for loderunner in command mode.

**Pull request merged**